text/0470-aurora-serverless-v2.md

```diff
@@ -0,0 +1,173 @@
+# Aurora Serverless v2 support
+
+* **Original Author(s):**: @pahud
+* **Tracking Issue**: #470
+* **API Bar Raiser**: @vinayak-kukreja
+
+Allow users to create Amazon Aurora Serverless v2 instances with the `aws-rds` module.
+
+## Working Backwards
+
+> This section should contain one or more "artifacts from the future", as if the
+> feature was already released and we are publishing its CHANGELOG, README,
+> CONTRIBUTING.md and optionally a PRESS RELEASE. This is the most important
+> section of your RFC. It's a powerful thought exercise which will challenge you
+> to truly think about this feature from a user's point of view.
+>
+> Choose *one or more* of the options below:
+>
+> * **CHANGELOG**: Write the changelog entry for this feature in conventional
+>   form (e.g. `feat(eks): cluster tags`). If this change includes a breaking
+>   change, include a `BREAKING CHANGE` clause with information on how to
+>   migrate. If migration is complicated, refer to a fictional GitHub issue and
+>   add its contents here.
+
+feat(rds): Aurora Serverless v2 support
+
+>
+> * **README**: If this is a new feature, write the README section which
+>   describes this new feature. It should describe the feature and walk users
+>   through usage examples and description of the various options and behavior.
+>
+
+## Aurora Serverless v2 support
+
+Aurora Serverless v2 is an on-demand, autoscaling configuration for Amazon Aurora.
+Aurora Serverless v2 helps to automate the processes of monitoring the workload and
```

Comment:
Would it make sense to have helper methods supporting [metrics](https://docs.aws.amazon.com/cdk/v2/guide/resources.html#resources_metrics) here?

Reply:
As DatabaseClusterV2 extends the [DatabaseClusterBase](https://github.com/aws/aws-cdk/blob/523cada6efe13ad0d1cae285c33bde814f57a226/packages/%40aws-cdk/aws-rds/lib/cluster.ts#L315-L368) abstract class and implement IDatabaseCluster so all helper methods defined in the base class should be available for DatabaseClusterV2. If we need to add additional helper methods for this new construct we can consider either add in the abstract class or this class. As this is a new construct without breaking changes, I think it's always possible to add any helper methods if necessary but doesn't have to be do that in the first PR.

Reply:
And Yes! I will add a section for all known use cases.

Reply:
Use Cases added.

Reply:
Thank you for adding those. 

In regards to the helper methods, what I want to convey is to think about how certain functions enable a better customer experience. They can be implemented later but the RFC would be a good place to list them out if any.